tests/e2e-cypress/integration/drawing/profile.cy.js

```diff
@@ -34,7 +34,7 @@ const testInfo = ['0.00m', '0.10m', '0.10m', "1'342m", "1'342m", '4.50m', '4.51m
 describe('Profile popup', () => {
     context('check how app behave with backend output', () => {
         const goToDrawingWithMockProfile = (mockValue) => {
-            cy.mockupBackendResponse('rest/services/profile.json', mockValue, 'profile')
+            cy.mockupBackendResponse('rest/services/profile.json?*', mockValue, 'profile')
```

Comment:
why not `cy.mockupBackendResponse('rest/services/profile.json**', mockValue, 'profile')`?
now if the app calls the backend without any URL argument (without the `?...` part of the URL) it might not be caught anymore, no?